_targets.R

```diff
@@ -12,39 +12,32 @@ tar_option_set(packages = c("tidyverse", "dataRetrieval")) # Loading tidyverse b
 
 
 p1_targets_list <- list(
-  # tar_target(
-  #   create_directories,
-  #   create_dirs(site_num = c("01427207", "01432160","01435000", "01436690", "01466500")),
-  #   format = "file"
-  # ),
   tar_target(
     nwis_01427207_data_csv,
-    download_nwis_site_data("01427207","nwis_01427207_data.csv"),
+    download_nwis_site_data_file("01427207","nwis_01427207_data.csv"),
     format = "file"
   ),
   tar_target(
     nwis_01432160_data_csv,
-    download_nwis_site_data("01432160","nwis_01432160_data.csv"),
+    download_nwis_site_data_file("01432160","nwis_01432160_data.csv"),
     format = "file"
   ),
   tar_target(
     nwis_01435000_data_csv,
-    download_nwis_site_data("01435000","nwis_01435000_data.csv"),
+    download_nwis_site_data_file("01435000","nwis_01435000_data.csv"),
     format = "file"
   ),
   tar_target(
-    nwis_01436690_data_csv,
-    download_nwis_site_data("01436690","nwis_01436690_data.csv"),
-    format = "file"
+    nwis_01436690_data,
+    download_nwis_site_data_object("01436690")
   ),
   tar_target(
-    nwis_01466500_data_csv,
-    download_nwis_site_data("01466500","nwis_01466500_data.csv"),
-    format = "file"
+    nwis_01466500_data,
+    download_nwis_site_data_object("01466500")
   ),
   tar_target(
     site_data_concat,
-    concat_files_to_df(c(nwis_01427207_data_csv, nwis_01432160_data_csv, nwis_01435000_data_csv, nwis_01436690_data_csv, nwis_01466500_data_csv))
+    concat_files_to_df(target_files = c(nwis_01427207_data_csv, nwis_01432160_data_csv, nwis_01435000_data_csv), target_object_1 = nwis_01436690_data, target_object_2 = nwis_01466500_data)
   ),
```

Comment:
Nice - while this setup of mixing file and object targets together probably won't be used a lot in practice it is good to see how you could use ones of both types. At this point if you wanted to convert all of them over to object types for consistency you are welcome to do that.